### PR TITLE
Add community information to README.md

### PR DESCRIPTION
**Pull request by JasonvanBrackel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
-# community
-Longhorn Community
+# Longhorn Community
+
+Focuses on supporting the Longhorn Cloud native distributed block storage built on and for Kubernetes
+
+## Meetings
+* Regular Meeting: [Every 2nd Friday of the Month at 12:00 Eastern Standard Time (EST)](https://zoom.us/j/710325604) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=12:00&tz=Eastern%20Standard%20Time%20%28EST%29).
```
Comment (yasker) [inline README.md]: Can we do PST and EST?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,56 @@
...
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1QD119jBX1h544ZQ9E4XaititEMfq8qEByx-42MtRC-g/).
+  * [Meeting recordings](https://www.youtube.com/channel/UCGk1f-LCVmccf1pX4OvF1Ig).
+
```
Comment (yasker) [inline README.md:8]: Do we have an easy way to add a calendar invitation?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,56 @@
...
+## Leadership
+
+### Chairs
+The Chairs of the group run operations.
+
+* Jason van Brackel (**[@jasonvanbrackel](https://github.com/JasonvanBrackel)**), Rancher Labs
+* TBD
```
Comment (yasker) [inline README.md]: I think we don't need `TBD` for now.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,56 @@
...
+
+### Technical Leads
+The Technical Leads establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Sheng Yang (**[@yasker](https://github.com/yasker)**), Rancher Labs
+* TBD
```
Comment (yasker) [inline README.md]: Same here.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,56 @@
...
+
+## Contact
+- Slack: [#longhorn-storage](https://rancher-users.slack.com/messages/longhorn-storage)
+- [Mailing list](https://groups.google.com/forum/#!forum/longhorn-io)
+- [Open Community Issues/PRs](https://github.com/longhorn/longhorn/issues)
+
+## Subprojects
```
Comment (yasker) [inline README.md]: I don't think we need this section for now. And those are not really separate projects.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,56 @@
...
+
+### longhorn-ui
+
+### longhorn-engine
+
+### longhorn-instance-manager
+
+## Getting Started
+
+If you're looking for information getting started with longhorn see [Longhorn](https://github.com/longhorn/longhorn)
```
Comment (yasker) [inline README.md]: `Longhorn` instead of `longhorn`

File: README.md [hunk 7]
```diff
@@ -1,2 +1,56 @@
...
+## Discuss
+Join us on [Slack](https://rancher-users.slack.com/messages/longhorn-storage) or the [Rancher Forums](https://forums.rancher.com/c/longhorn) to chat about Longhorn.
+
+## Contributing
+
+To get yourself familiar with contributing, please read the Please read [Code of Conduct](https://github.com/longhorn/longhorn/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/longhorn/longhorn/blob/master/CONTRIBUTING.md) before contributing.
+
+As always, you can also find the active list of open issues and proposals [Open Code Issues and PRs for Longhorn](https://github.com/longhorn/longhorn/issues) on GitHub.
+
+## Meeting Recordings
```
Comment (yasker) [inline README.md:44]: Already mentioned above?

File: README.md [hunk 8]
```diff
@@ -1,2 +1,56 @@
-# community
-Longhorn Community
+# Longhorn Community
+
+Focuses on supporting the Longhorn Cloud native distributed block storage built on and for Kubernetes
```
Comment (yasker) [inline README.md]: Not really necessary?... It's pretty obvious.

**Review by yasker — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (JasonvanBrackel) [reply README.md:44]: Leaving this as is as it's common practice in K8s sigs

File: README.md (hunk 8 above)
Comment (JasonvanBrackel) [reply README.md]: This is necessary.  Without it the file is not valid markdown

Comment (yasker) [reply README.md]: I meant `Focuses on supporting the Longhorn Cloud native distributed block storage built on and for Kubernetes`

File: README.md (hunk 2 above)
Comment (yasker) [reply README.md:8]: The link `https://raw.githubusercontent.com/longhorn/community/master/Longhorn%20Community%20Meetup.ics` opens up a raw file on my computer. People may be confused by it. Not sure if we can get something like https://goo.gl/eyutah

**Review by yasker — CHANGES_REQUESTED** [review]

**Comment by yasker** [comment]
@JasonvanBrackel Btw, missing the `Signed-off-by`.

**Review by yasker — APPROVED** [review]

**Pull request merged**